collector/src/main/java/io/prometheus/jmx/JmxCollector.java

```diff
@@ -350,7 +350,7 @@ public static void main(String[] args) throws Exception {
       + "`hostPort`: `" + hostPort + "`,"
       + "}").replace('`', '"'));
       for(MetricFamilySamples mfs : jc.collect()) {
-        System.out.println(mfs);
+          LOGGER.info(mfs != null? mfs.toString() : null);
```

Comment:
This is a command line tool for debugging and development, stdout is appropriate there.